.travis.yml

```diff
@@ -1,30 +1,31 @@
 languages: java
 # TODO: Add openjdk8 once it is available in Travis.
 jdk:
-  - oraclejdk8
+- oraclejdk8
 sudo: false
-script: ./gradlew clean build coveralls --scan --no-daemon --info
+script: ./gradlew clean build --scan --no-daemon --info
 
 # see https://docs.travis-ci.com/user/languages/java/#Projects-Using-Gradle
 before_cache:
-  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
-  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+- rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+- rm -fr $HOME/.gradle/caches/*/plugin-resolution/
 cache:
   directories:
-    - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
+  - $HOME/.gradle/caches/
+  - $HOME/.gradle/wrapper/
 
 notifications:
   email:
     recipients:
-      - azkaban-dev@linkedin.com
+    - azkaban-dev@linkedin.com
     on_success: never
     on_failure: always
 
 env:
   global:
-    # Reduce the memory pressure on Travis CI to reduce build failures.
-    - GRADLE_OPTS="-Xms128m"
+  # Reduce the memory pressure on Travis CI to reduce build failures.
+  - GRADLE_OPTS="-Xms128m"
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
+- ./gradlew coveralls --no-daemon
```

Comment:
How does this change reduce the log output?

Reply:
Is it because it no longer runs with the --info option?

Reply:
Correct -- no `--info` for `coveralls` task. If anyone knows a better way to change logging level for just one task in a bigger `gradle` run, please let me know -- this could shave a few seconds off the execution time.